packages/cds-odata-annotation-converter/src/transforms/annotation/visitor-state.ts

```diff
@@ -7,6 +7,7 @@ export interface Context {
      */
     recordType?: string;
     termType?: string;
+    isFlattened$Type?: boolean;
```

Comment:
This should not be needed, because $Type is a leaf node and we don't need to pass context deeper.

Reply:
removed